cnv/cnv_release_notes/cnv-release-notes.adoc

```diff
@@ -66,6 +66,14 @@ Virtualization Validation Program (SVVP) to run Windows Server workloads.
 ** SVVP Certificate Red Hat Enterprise Linux CoreOS 8 workers are named _Red Hat OpenShift
 Container Platform 4 on RHEL CoreOS 8_.
 
+=== Web console
+// CNV-4076 UI view/config dedicated resources
+* A new option to view and configure dedicated resources for virtual machines is
+available in the web console *Virtual Machine Details* view. When you enable dedicated
+resources, you ensure that your virtual machine's workload runs on CPUs that
+are not used by other processes. This improves your virtual machine's performance
```

Comment:
Previous versions had "This *can* improve VM performance", which I think is important to retain as I don't believe an improvement is guaranteed.

Reply:
Done.